internal/hmyapi/blockchain.go

```diff
@@ -112,6 +112,12 @@ func (s *PublicBlockChainAPI) BlockNumber() hexutil.Uint64 {
 	return hexutil.Uint64(header.Number().Uint64())
 }
 
+// ResendTx retrieves blockHash by TxID and add blockHash to CxPool for resending
+func (s *PublicBlockChainAPI) ResendTx(ctx context.Context, txID common.Hash) (bool, error) {
```

Comment:
```suggestion
func (s *PublicBlockChainAPI) ResendCx(ctx context.Context, txID common.Hash) (bool, error) {
```

Reply:
By the way, is resending egress CX receipts a responsibility of a blockchain API proper?  Egress CX sending is an implementation detail of a cross-shard transaction, which is at the protocol level…  Maybe we need to revisit elsewhere to root this.